explore_utils.py

```diff
@@ -56,3 +56,29 @@ def find_most_common_traffic_sources(dataset,num=5):
 
     """
     return dataset.train['trafficSource.source'].value_counts().head(num)
+
+
+def find_one_visit_percent(dataset):
+    """Finds the percent of visitors to the store that only visit once
+
+    args:
+        dataset (Dataset): the google analystics dataset
+    
+    returns:
+        The percent of total customers that have only visited once, based on their ID
+    
+    """
+    data = dataset.train.copy()
+    
+    #gets DataFrame of each visitor's total number of visits
+    total_visits = data.groupby("fullVisitorId")['visitNumber'].sum()
```

Comment:
I don't think this is causing a problem, but it might not be doing exactly what you think. Summing over `visitNumber`, isn't the same as the total number of visits. For example, if someone visited 5 times, their visit numbers should be 1, 2, 3, 4, 5. The sum of these is 1 + 2 + 3 + 4 + 5 = 15, not the 5 that you would expect. I think it should be `max()` instead of `sum()`.

Still approving because this doesn't affect the case where total_visits == 1.